### DISCUSSION TITLE
Disable conversion/reformatting of input values (date, time, ?)

### DISCUSSION THREAD

**Discussion by mcint** [freeplane/freeplane#2138] [Help]
I've repeatedly found myself fighting or working around the convenience features that parse, at least date and time.  I want to enter other values, with similar separators, or enter fractions, and find the values converted to dates. 

- `1/1` -> `1/1/24`
- `1:0` -> `12/7/24, 1:00 AM`

I would love to disable this. Any idea if it could easily be made disable-able? Do others feel the same?

**Comment by euu2021** [comment]
https://docs.freeplane.org/user-documentation/Data_recognition_and_data_formats.html?highlight=parse#how-to-switch-data-recognition-off
